### ISSUE TITLE
Cannot build v. 0.8.19

### ISSUE THREAD

**Issue by CoolMind** [vector-im/riot-android#2753]
After many attempts of building the project and updating libraries I write here. Most problems were with [android-ktx](https://github.com/android/android-ktx). It was archived and `toast()` method was removed. I had to replace some `com.android.support` with `androidx.support`  and downgrade to `'androidx.core:core-ktx:1.0.0-beta01'`. 

A very annoying error during building made me remove that library:

> 	Attribute application@appComponentFactory value=(androidx.core.app.CoreComponentFactory) from [androidx.core:core:1.0.0] AndroidManifest.xml:22:18-86
> 	is also present at [com.android.support:support-compat:28.0.0] AndroidManifest.xml:22:18-91 value=(android.support.v4.app.CoreComponentFactory).
> 	Suggestion: add 'tools:replace="android:appComponentFactory"' to <application> element at AndroidManifest.xml:4:5-17:19 to override.
> 

After that I could replace all suspecting methods.

I changed all `toast()` in the project with `Toast.makeText(this@AbstractWidgetActivity, errorMessage, Toast.LENGTH_SHORT).show()` (change class name).

It has to be done about hundred substitutions more.

**Comment by bmarty** [comment]
Feel free to open a PR with all your fixes

**Comment by CoolMind** [comment]
@bmarty, thanks! You are right. Sorry, I decided to cancel this issue, because we performed our test and currently not needed to develop further. After several hours of attempts I started from the beginning. When AS asked to update Gradle to 3.2.1, I refused. So, I haven't have to update Android libraries to 28.0.0 and remove AndroidX. Then I fixed 3 lines in code, that compiler marked as errors. Then compiled the project again.
